build.sbt

```diff
@@ -66,6 +67,7 @@ libraryDependencies ++= Seq(
   "org.scalaz" % "scalaz-core_2.11" % "7.1.3",
   "com.github.pathikrit" %% "better-files" % "2.13.0",
   "org.liquibase" % "liquibase-core" % "3.4.2",
+  "com.squants"  %% "squants"  % "0.6.1-SNAPSHOT",
```

Comment:
Instead of this, you could use the memory parsing library that's included in wdl4s.  Specifically the [MemoryUnit](https://github.com/broadinstitute/wdl4s/blob/develop/src/main/java/wdl4s/parser/MemoryUnit.java) and [MemorySize](https://github.com/broadinstitute/wdl4s/blob/develop/src/main/scala/wdl4s/MemorySize.scala).  [The tests](https://github.com/broadinstitute/wdl4s/blob/develop/src/test/scala/wdl4s/MemorySizeSpec.scala) have some examples of how to use it.

Squants looks like it does the job too... though it is perhaps a little concerning to have to use a snapshot release.

I guess I'm less concerned with which library is used and more concerned that we how have two of these libraries.  What do you think about choosing one?  I imagine we would still need at least a small piece of `MemorySize`, specifically the `parse` method so it can regex match and split memory strings we get from WDL runtime sections